mit.md

```diff
@@ -0,0 +1 @@
+MIT license, A software license tells others what they can and can't do with your source code
```

Comment:
Kindly refer to this [MIT.md](https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt) to fill your MIT.md document